bitcoin/CHANGELOG.md

```diff
@@ -1,3 +1,46 @@
+# Unreleased (changelog done for PRs merged as of June 24)
+
+- Bump MSRV to Rust 1.48.0 [#1729](https://github.com/rust-bitcoin/rust-bitcoin/pull/1729)
+
+- API Improvements:
+  - [Add kilo weight unit conversion](https://github.com/rust-bitcoin/rust-bitcoin/pull/1735)
+  - [Add ChainHash::from_genesis_block_hash](https://github.com/rust-bitcoin/rust-bitcoin/pull/1753)
+  - [Add implementation of PartialEq trait for Address with NetworkUnchecked](https://github.com/rust-bitcoin/rust-bitcoin/pull/1757)
+  - Add [unsafe address type conversions](https://github.com/rust-bitcoin/rust-bitcoin/pull/1765)
+  - [Add `Inventory::network_hash()` method](https://github.com/rust-bitcoin/rust-bitcoin/pull/515)
+  - [Rename `Transaction::is_coin_base` to `is_coinbase`](https://github.com/rust-bitcoin/rust-bitcoin/pull/1796)
+  - [Implement `serde::Serialize` for `Address`](https://github.com/rust-bitcoin/rust-bitcoin/pull/1785)
+  - [Expose valid (min, max) difficulty transition thresholds](https://github.com/rust-bitcoin/rust-bitcoin/pull/1820)
+  - Add functionality to [generate PrivateKey](https://github.com/rust-bitcoin/rust-bitcoin/pull/1837)
+  - [Allow parsing sub-sat denominations with decimal points](https://github.com/rust-bitcoin/rust-bitcoin/pull/1768)
+  - [Fix associated constants of `InputWeightPrediction`](https://github.com/rust-bitcoin/rust-bitcoin/pull/1835)
+  - [Add a checked version of weight mul fee_rate](https://github.com/rust-bitcoin/rust-bitcoin/pull/1864)
+  - [Use `Amount` type for `TxOut` value field](https://github.com/rust-bitcoin/rust-bitcoin/pull/1811)
+  - Add support for [alloc-free parse errors](https://github.com/rust-bitcoin/rust-bitcoin/pull/1297)
+  - [Rename `TaprootSpendInfo::as_script_map` `to script_map`](https://github.com/rust-bitcoin/rust-bitcoin/pull/1897)
+  - [Add `Amount::from_int_btc`](https://github.com/rust-bitcoin/rust-bitcoin/pull/1870)
+  - Make [`ckd_priv` function private](https://github.com/rust-bitcoin/rust-bitcoin/pull/1882) and impl `AsRef` for `ChildNumber`.
+  - [Use hex_lit::hex in benches](https://github.com/rust-bitcoin/rust-bitcoin/pull/1941) (backported to v0.30.1)
+  - [Add a verify function to PublicKey](https://github.com/rust-bitcoin/rust-bitcoin/pull/1911)
+  - [script: Add method `count_sigops`](https://github.com/rust-bitcoin/rust-bitcoin/pull/1890)
+    - [Implement `From<PublicKey> for XOnlyPublicKey`](https://github.com/rust-bitcoin/rust-bitcoin/pull/1901)
```

Comment:
nit: Indentation.

Reply:
oh, lol, this does actually affect the rendered output. Good catch.

Reply:
Fixed, thanks.